user/pull-requests.md

```diff
@@ -84,3 +66,14 @@ The command "eval git fetch origin +refs/pull/994/merge: " failed
 ```
 
 Restoring the branch of a merged pull request will not trigger a build, nor will pushing a new commit to a branch that has already been merged.
+
+## 'Double builds' on pull requests
+
+If you see two build status icons on your GitHub pull request, it means there is one build for the branch, and one build for the pull request itself (actually the build for the merge of the head branch with the base branch specified in the pull request).
+
+[Build pushed branches](/user/web-ui/#build-pushed-branches) and [Build pushed pull requests](/user/web-ui/#build-pushed-pull-requests) control this behaviour.
```

Comment:
Nitpick: Are we standardizing on American or British spelling?

Reply:
These links are dead. Are they meant to be added with this PR, or are they meant to point to something else? I looked around, but this wasn't clear to me.

Reply:
https://github.com/travis-ci/docs-travis-ci-com/pull/1682#issuecomment-362588202 mentions these

Reply:
In fact, @BanzaiMan if you wanna take a look at that PR as well... we can merge that one then this one.

Reply:
On the spelling issue, that is a ball I've been avoiding. :-p Pretty sure we should be using American spelling, but I've not been putting huge effort into checking that.

There should be a linter for this, no?